placeholders.go

```diff
@@ -25,6 +26,17 @@ func parsePlaceholders(input string, r *http.Request) string {
 			input = strings.Replace(input, "{host}", r.URL.Host, -1)
 		case "{hostonly}":
 			input = strings.Replace(input, "{hostonly}", r.URL.Hostname(), -1)
+		case "{label":
```

Comment:
Since it's a dynamic placeholder, ``case`` is not gonna work here. You can use an if statement and ``strings.HasPrefix`` or something else after the switch case.

Reply:
Thanks - I've moved that check outside of the switch statement